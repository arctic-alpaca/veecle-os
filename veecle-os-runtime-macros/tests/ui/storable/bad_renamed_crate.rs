#[derive(Debug, veecle_os_runtime_macros::Storable)]
#[storable(crate = "::veecle_os_runtime")]
pub struct Sensor0 {
    test: u8,
}

#[derive(Debug, veecle_os_runtime_macros::Storable)]
#[storable(crate())]
pub struct Sensor1 {
    test: u8,
}
#[derive(Debug, veecle_os_runtime_macros::Storable)]
#[storable(crate = ::veecle_os_runtime, crate = ::veecle_os_runtime2)]
pub struct Sensor2 {
    test: u8,
}

fn main() {}
