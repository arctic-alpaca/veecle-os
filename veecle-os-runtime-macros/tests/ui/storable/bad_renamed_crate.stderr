error: Unexpected meta-item format `list`
 --> tests/ui/storable/bad_renamed_crate.rs:8:12
  |
8 | #[storable(crate())]
  |            ^^^^^

error: Duplicate field `crate`
  --> tests/ui/storable/bad_renamed_crate.rs:13:41
   |
13 | #[storable(crate = ::veecle_os_runtime, crate = ::veecle_os_runtime2)]
   |                                         ^^^^^
